litestar/openapi/plugins.py

```diff
@@ -0,0 +1,694 @@
+from __future__ import annotations
+
+from pathlib import Path
+from typing import TYPE_CHECKING, Any, Sequence
+
+import httpx
+import msgspec
+import yaml
+
+from litestar.enums import MediaType, OpenAPIMediaType
+from litestar.handlers import get
+from litestar.serialization import encode_json, get_serializer
+
+if TYPE_CHECKING:
+    from litestar.connection import Request
+    from litestar.router import Router
+
+__all__ = (
+    "OpenAPIRenderPlugin",
+    "RapidocRenderPlugin",
+    "RedocRenderPlugin",
+    "ScalarRenderPlugin",
+    "StoplightRenderPlugin",
+    "SwaggerRenderPlugin",
+    "YamlRenderPlugin",
+)
+
+_favicon_url = "https://raw.githubusercontent.com/litestar-org/branding/main/assets/Branding%20-%20PNG%20-%20Transparent/Badge%20-%20Blue%20and%20Yellow.png"
```

Comment:
Maybe we point this at a CDN as well? e.g.,
Host it as is, put generate the CDN url for end user redundancy through https://www.jsdelivr.com/github.
`https://cdn.jsdelivr.net/gh/litestar-org/branding@main/assets/Branding%20-%20PNG%20-%20Transparent/Badge%20-%20Blue%20and%20Yellow.png`